examples/vadd/Makefile

```diff
@@ -0,0 +1,72 @@
+F90 ?= gfortran
+
+
+KERNELS = vadddevice.cl
+
+
+# Options: JIT | XILINX_FPGA | INTEL_FPGA
```

Comment:
Please extend comment to say this is picked up by an #ifdef in the source code.

Reply:
Done